utils/jsonConverter.js

```diff
@@ -0,0 +1,157 @@
+// import exportfromJSON from 'export-from-json';
```

Comment:
this looks fine for now because we will probably change this a lot further down the road

Reply:
but we should definitely do a code clean up once this feature is more finalized